tests/stdlib/tosenv.nim

```diff
@@ -1,22 +1,41 @@
 discard """
   matrix: "--threads"
   joinable: false
-  target: "c cpp"
+  targets: "c cpp"
 """
+import os
 
-import os, system
+block delEnv:
+    const dummyEnvVar = "DUMMY_ENV_VAR" # This env var wouldn't be likely to exist to begin with
+    doAssert existsEnv(dummyEnvVar) == false
+    putEnv(dummyEnvVar, "1")
+    doAssert existsEnv(dummyEnvVar) == true
```

Comment:
```suggestion
    doAssert existsEnv(dummyEnvVar)
```

ditto, `doAssert foo == false` => `doAssert not foo` is more idiomatic

Reply:
Sure, this is the way it was in tos.nim but I can change it to be more idiomaatic